barcodes_generator_abstract/models/barcode_generate_mixin.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2014-TODAY GRAP (http://www.grap.coop)
+# Copyright (C) 2016-TODAY La Louve (http://www.lalouve.net)
+# Copyright 2017 LasLabs Inc.
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import _, api, exceptions, fields, models
+
+from .barcode_rule import _GENERATE_TYPE
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import barcode
+except ImportError:
+    _logger.debug("Cannot import 'viivakoodi' python library.")
+    barcode = None
+
+
+class BarcodeGenerateMixin(models.AbstractModel):
+
+    _name = 'barcode.generate.mixin'
```

Comment:
Add `_description` for avoiding runbot warning

Reply:
About the library, tell me if I should change it
About _description, I will do a force push tomorrow

Reply:
@legalsylvain @yucer what do you think?

Reply:
Hi. As said in #193, I have no point of view, regarding PIL / Pillow neither regarding the quality of the code of both librairies.
But the othe lib has more contributors. So I'm in favour to switch, if it works correctly. 

@cvinh, could you test and check if all is OK ? 

regards.

Reply:
@pedrobaeza could you help with the runbot error ? I don't see where it fails... thanks in advance

Reply:
@gurneyalex could you help with the runbot error ? I don't see where it fails... thanks in advance

Reply:
@pedrobaeza sorry for the late response. I am not too much familiarized with these modules. Just noted that there were two alternate python libraries for the same feature, and the one not used had more activity.

Thanks for the feed back.

Reply:
Module was already merge, so a bit late I'm afraid. Any way, we have switched the library.